src/Symfony/Component/Form/Extension/Core/DataTransformer/BooleanToStringTransformer.php

```diff
@@ -24,12 +24,16 @@ class BooleanToStringTransformer implements DataTransformerInterface
 {
     private $trueValue;
 
+    private $falseValues;
+
     /**
-     * @param string $trueValue The value emitted upon transform if the input is true
+     * @param string $trueValue   The value emitted upon transform if the input is true
+     * @param array  $falseValues
      */
-    public function __construct(string $trueValue)
+    public function __construct(string $trueValue, array $falseValues = array(null))
     {
         $this->trueValue = $trueValue;
+        $this->falseValues = $falseValues;
```

Comment:
IMO we should also add some error handling in case `$trueValue` is one of the elements in `$falseValues`

Reply:
Is InvalidArgumentException ok?

Reply:
yes